rotkehlchen/tests/api/test_assets.py

```diff
@@ -76,13 +81,29 @@ def test_query_owned_assets(
             id=-1,
             asset=A_EUR,
             label='My EUR bank',
-            amount=FVal('1550'),
+            amount=FVal(1550),
```

Comment:
it's fine to do this change, but why did you do it here and now?

Reply:
just because I was editing the test and saw it